devops/wsi_deid/provision.py

```diff
@@ -49,6 +49,7 @@ def provision():
         PluginSettings.HUI_FINISHED_FOLDER: ('Approved', True),
         PluginSettings.HUI_REPORTS_FOLDER: ('Reports', True),
         PluginSettings.WSI_DEID_SCHEMA_FOLDER: ('Schema', True),
+        PluginSettings.WSI_DEID_DISABLED_SCHEMA_FOLDER: ('Disabled', True),
```

Comment:
Hmm... I was thinking this wouldn't be a setting; we would just create the Disabled folder as a side effect of the schema folder (i.e., we'd add it as a special case when adding the schema folder).

Reply:
Ok, I'll update

Reply:
changed in 0b63bc92a31ac6eddd35222daba11279cf2439d6